manifests/calico-vxlan.yaml

```diff
@@ -5,4 +5,5 @@ layout: null
 datastore: kubernetes
 network: calico
 vxlan: true
+eks: true
```

Comment:
I think all the places we use this aren't actually EKS specific settings. Maybe this should be called "amazon"?